c2corg_ui/templates/outing/filters.html

```diff
@@ -1,6 +1,9 @@
 <%!
-from c2corg_common.attributes import default_langs, quality_types, activities, frequentation_types, \
-    condition_ratings, glacier_ratings, avalanche_signs
+from c2corg_common.attributes import default_langs, quality_types, activities, \
+    frequentation_types, condition_ratings, glacier_ratings, avalanche_signs, \
+    climbing_ratings, ski_ratings, global_ratings, engagement_ratings, equipment_ratings, \
```

Comment:
Again I dont see why climbing_ratings is used

Reply:
It's used at https://github.com/c2corg/v6_ui/blob/d5ef793673d790adffcea735525ce304f504b971/c2corg_ui/templates/outing/filters.html#L123 for ``rock_free_rating``.
Several ratings use the same climbing_ratings values defined at https://github.com/c2corg/v6_common/blob/master/c2corg_common/attributes.py#L224

Reply:
roger that